src/main/api/ledger/bitcoin/transaction.ts

```diff
@@ -85,7 +85,31 @@ export const send = async ({
         addressFormat === AddressFormat.P2TR ? tapRootDerivationPaths : getDerivationPaths(walletAccount, network),
       network: network
     })
-    const txHash = await clientLedger.transfer({ walletIndex, asset: AssetBTC, recipient, amount, memo, feeRate })
+
+    const fee = await clientLedger.getFeesWithRates({ sender, memo })
+    const feeAmount = fee.fees[feeOption]
+    const feeRate = fee.rates[feeOption]
+    const bal = await clientLedger.getBalance(sender)
+    const btcBalance = bal[0]
+    const transactionSize = feeAmount.amount().toNumber() / feeRate
+    const roundedFeeRate = Math.ceil(feeRate)
+    const adjustedFee = baseAmount(roundedFeeRate * transactionSize)
+    const feeValue = adjustedFee.amount().toNumber()
+    const roundedFeeValue = Math.ceil(feeValue / 1000) * 1000
```

Comment:
Why is this line here and not in BCH below? I assume they should be the same, as with other UTXOs like LTC.

Reply:
cus with bch I don't need to round the fee up.